docs/events/wg_workshops/2023-06-28-june-meeting/opensafely.md

```diff
@@ -0,0 +1,115 @@
+# OpenSAFELY community presentation/discussion notes
+
+Speaker: Seb Bacon (Bennett Institute)
+
+- OpenSAFELY is an open software, not a database
+- Give researchers access to a huge amount of data through TPP & EMIS
+
+  - 58 million patients' GP data
+  - &gt; 70bn rows of primary care data
+  - Key health datasets, e.g. death register, A&E, hospital cases etc.
+  - Research datasets, e.g. ONS, ...
+
+- Not a database
+
+- Installed on top of for example TPP and EMIS (GP health record system providers)
+
+  - TPP and EMIS are the ones running the database and providing safe infrastructure
+
+- How did it gain rights/earned trust to manage this data?
+
+- OpenSAFELY addressed 4 core problems:
+  - Privacy
+  - Transparency
+  - Reproducibility
+  - ....
+- OpenSAFELY is giving access to data at arms length
+- As a researcher you don't do data curation on the real data, you do it on randomly generated ~~synthetic~~ data (not synthetic, for reasons not explained today)
+  - Code can be generated and tested to ensure it will work on real data
+  - Code is run within SDE separate to the researcher
+    - Researcher never has access to the SDE
+  - Code for the platform is online, fully tested and documented
+  - Every bit of research that has been run with sensitive data on the platform has to be open by design
+    - Code can't be run unless it can be taken from GitHub
+  - Earned trust through transparency
+    - Latest job requests logs openly available
+      - Inc info on project, who ran job, how long it ran for etc.
+    - Linked from dashboard to code that was run
+    - All outputs that have been output checked as safe to release viewable on the dashboard
+- Code is rarely shared between projects
+- Using the SNOMED codes but hard to know if these are as accurate as can be
+- ehrQL: Special language created by OpenSafely for data curation
+- Federated analysis: single codebase, executed in multiple locations. Moves code to data.
+
+## Questions
+
+Synthetic data:
+
+- Is the synthetic data generating process published anywhere?
+- Can OpenSAFELY produce synthetic data at event level?
+- What's the difference between fake data and synthetic data?
+  - Ed: I think synthetic data will always be something that retains some properties of the original data, because it has been generated _from_ the real data, using some kind of synthetic data generation algorithm. Fake data is just random, but structured in the same way as real data
```

Comment:
```suggestion
  - I think synthetic data will always be something that retains some properties of the original data, because it has been generated _from_ the real data, using some kind of synthetic data generation algorithm. Fake data is just random, but structured in the same way as real data
```

Reply:
Most of the answers are either from Seb, or based on what Seb said. This is the only answer from someone else, so I think we should make that clear.